src/Nethermind/Nethermind.Facade/Eth/EthSyncingInfo.cs

```diff
@@ -55,18 +63,33 @@ public SyncingResult GetFullInfo()
             if (_syncConfig.FastSync)
             {
                 if (_syncConfig.DownloadReceiptsInFastSync &&
-                    _receiptStorage.LowestInsertedReceiptBlockNumber > _syncConfig.AncientReceiptsBarrierCalc)
+                    (_receiptStorage.LowestInsertedReceiptBlockNumber == null || _receiptStorage.LowestInsertedReceiptBlockNumber > _syncConfig.AncientReceiptsBarrierCalc))
```

Comment:
previous condition was fine, null will not be > than number.

Reply:
@deffrian wrote some unit tests

Reply:
But we want to have `true` if `LowestInsertedReceiptBlockNumber` is `null` and this is the reason of this change

Reply:
Then we need to check if we are in FasSync. In FullSync this will always be null!

Reply:
`if (_syncConfig.FastSync)` is 3 lines above